test/ui_test_label_studio.py

```diff
@@ -0,0 +1,213 @@
+        action.move_to_element_with_offset(self.driver.find_element_by_tag_name('body'), 0,0)
+        #
+        # el=self.driver.find_element_by_xpath('/html/body/div/div/div/div/div/div/div[1]/div/div/div[1]/div[2]/div/canvas')
+        # action = webdriver.common.action_chains.ActionChains(self.driver)
+        # action.move_to_element_with_offset(el, 500, 500)
+        # action.click()
+        # action.perform()
+        #
+        x = 700
+        y = 300
+        action.move_by_offset(x, y).click().perform()
+        action.move_by_offset(x, y+50).click().perform()
+        action.move_by_offset(x+30, y+30).click().perform()
+        action.move_by_offset(x, y).click().perform()
+        #
+        #Go to outsite from iframe
+        self.driver.switch_to_default_content()
+
+
+    #Step2 - Import tasks to project
+    def step2(self, testCaseName):
+        # Go to page Import
+        print("..... Going to page Import .....")
+        self.driver.find_element_by_xpath(self.welcome_Import).click()
+        #
+        # Choose image for dragAndDrop
+        print('Direktory is: ' + str(os.getcwd()))
+        self.import_img_src = str(os.getcwd()) +'/'+ self.import_img_src
+        print('Image is: ' +str(self.import_img_src))
+        self.driver.find_element_by_id(self.import_img_draganddrop_id).send_keys(self.import_img_src)
+        print("Image choosed from system and placed into dragAndDrop area")
+        #
+        # Close 'Import status'
+        import_status_close_btn = self.wait.until(EC.element_to_be_clickable((By.ID, "upload-done-button")))
+        import_status_close_btn.click()
+        print("Popup window with the name 'Import status' were closed ")
+        # time.sleep(2)
+
+
+    #Step3 - Start labeling tasks
+    def step3(self, testCaseName):
+        # Go to page Start
+        print("..... Going to page Start .....")
+        self.driver.find_element_by_xpath(self.welcome_Start).click()
+        #
+        # Click on Label Airplane
+        # self.driver.find_element_by_xpath(self.start_label_car).click()
+        label_car_btn = self.wait.until(EC.element_to_be_clickable((By.CLASS_NAME, self.start_label_car)))
+        label_car_btn.click()
+        #
+        # Doing 3 clicks for create area
+        actions = ActionChains(self.driver)
+        x = 50
+        y = 150
+        actions.move_to_element_with_offset(self.driver.find_element_by_tag_name('body'), 0,0)
+        actions.move_by_offset(x, y).click().perform()
+        actions.move_to_element_with_offset(self.driver.find_element_by_tag_name('body'), 0,0)
+        actions.move_by_offset(x, y+50).click().perform()
+        actions.move_to_element_with_offset(self.driver.find_element_by_tag_name('body'), 0,0)
+        actions.move_by_offset(x+50, y+30).click().perform()
+        actions.move_to_element_with_offset(self.driver.find_element_by_tag_name('body'), 0,0)
+        actions.move_by_offset(x, y).click().perform()
```

Comment:
this can be simplified with just `move_by_offset` after each other, without resetting to body. also it's better to offset from image/canvas tag, it's more safe and logical.

Reply:
Without resetting we will have not wanted random dotes. 
Than is a good suggestion to offset by image/canvas, but when I tried offset with canvas: not succeeded.
Lets keep this implementation for now)

Reply:
where do these random dots can come from? you'll only make a few clicks with explicitly given offsets.
about canvas... there can be some hidden ones maybe, so this should be checked on frontend side

Reply:
This is working chain. We can discuss about needed scenarios and add them here)